envoy/filter/config_provider_manager.h

```diff
@@ -56,7 +56,7 @@ template <class FactoryCb> class FilterConfigProviderManager {
   /**
    * Get the stat prefix for the scope of the filter provider manager.
    */
-  virtual absl::string_view statPrefix() const PURE;
+  virtual const absl::string_view& statPrefix() const PURE;
```

Comment:
No need to return const-ref of absl::string_view. Returning `absl::string_view` is sufficient.

Reply:
ACK